pm64/src/bgm/bgm_ron.rs

```diff
@@ -0,0 +1,655 @@
+use std::{collections::HashMap, mem::MaybeUninit};
```

Comment:
I can't accept a near identical copy of bgm.rs as I'd need to maintain both forever. It also makes it hard to see what you've changed unless I diff the two files.